datasetComparison/src/test/scala/za/co/absa/hermes/datasetComparison/DatasetComparisonConfigSuite.scala

```diff
@@ -14,21 +14,35 @@
  * limitations under the License.
  */
 
-package za.co.absa.hermes.datasetComparison.config
+package za.co.absa.hermes.datasetComparison
 
 import org.scalatest.FunSuite
 
-class TypeSafeConfigSuite extends FunSuite {
+class DatasetComparisonConfigSuite extends FunSuite {
+  test("Manual Config Correct") {
+    val conf = DatasetComparisonConfig("errCol", "_actual", "_expected", false)
+    assert(conf.validate().isSuccess)
+    assert("errCol" == conf.errorColumnName)
+    assert("_actual" == conf.actualPrefix)
+    assert("_expected" == conf.expectedPrefix)
+    assert(!conf.allowDuplicates)
+  }
+
+  test("Manual Config Bad Column Name") {
+    val conf = DatasetComparisonConfig("errCol", "_actua l", "_expected", false)
+    assert(conf.validate().isFailure)
```

Comment:
Perhaps the checking that validation failure occurs as expected could be done for all the fields
```scala
   assert(DatasetComparisonConfig("errCol", "_actua l", "_expected", false).validate().isFailure)
    assert(DatasetComparisonConfig("er{r}Col", "_actual", "_expected", false).validate().isFailure)
    assert(DatasetComparisonConfig("errCol", "_actual", "_expe=cted", false).validate().isFailure)
```